tests/integration/test_bayesian_optimization.py

```diff
@@ -361,23 +355,86 @@ def test_bayesian_optimizer_with_deep_ensemble_finds_minima_of_scaled_branin(
     )
 
 
+@random_seed
```

Comment:
Have you manually run the develop-checks job for your branch to see whether this passes?